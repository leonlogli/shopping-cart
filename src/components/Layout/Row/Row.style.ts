import styled from 'styled-components'
import { Row } from 'styled-bootstrap-grid'

export const StyledRow = styled(Row)``

export default StyledRow
