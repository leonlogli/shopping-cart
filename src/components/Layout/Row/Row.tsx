import React from 'react'
import { RowProps as BaseContainerProps } from 'styled-bootstrap-grid'

import StyledRow from './Row.style'

export type RowProps = BaseContainerProps & {
  children: React.ReactNode
}

const Row = ({ children, ...other }: RowProps) => (
  <StyledRow {...other}>{children}</StyledRow>
)

export { Row }
export default Row
