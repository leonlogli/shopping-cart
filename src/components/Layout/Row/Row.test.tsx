import React from 'react'
import { mount } from 'enzyme'
import Row from './index'

describe('<Row />', () => {
  const wrapper = mount(
    <Row>
      <span>test</span>
    </Row>
  )

  it('should have the correct children', () => {
    expect(wrapper.find('span').length).toEqual(1)
  })

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
