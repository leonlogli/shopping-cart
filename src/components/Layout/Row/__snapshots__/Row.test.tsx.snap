// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> should match a snapshot 1`] = `
<Row>
  <Styled(styled.div)>
    <div
      className="cwNyiz"
      data-name="row"
    >
      <span>
        test
      </span>
    </div>
  </Styled(styled.div)>
</Row>
`;
